src/bun.js/api/bun/h2_frame_parser.zig

```diff
@@ -2526,3 +2521,11 @@ pub const H2FrameParser = struct {
         this.deinit();
     }
 };
+
+pub fn createNodeHttp2Binding(global: *JSC.JSGlobalObject) JSC.JSValue {
+    return JSC.JSArray.createComptime(global, .{
```

Comment:
Is this `JSC.JSArray.createComptime` being called not-at-comptime?

Reply:
this should get a clearer name

it is inferring the length at comptime and doing an `inline for` over it.

Reply:
(learning): From a zig POV how can it comptime the `items.len` when `items` is an `anytype` not known at comptime. Is it because `JSC.JSArray.createComptime` is designated as `inline` and so inlined at comptime and because `items` are fixed length literals?

Reply:
it is seen as a fixed-size array or tuple in this usage, so it knows there are two items. it doesn't know the values of this at compile time, but it knows the count.